bin/internal/shared.bat

```diff
@@ -157,4 +157,7 @@ GOTO :after_subroutine
   REM Exit Subroutine
   EXIT /B
 
+:final_exit
```

Comment:
What's the reason for moving this here?

Reply:
This is actually a bug, we can't jump to this label from shared.bat